bin/run_tests.py

```diff
@@ -42,12 +42,15 @@
         sys.executable,
         "-m",
         "pytest",
+        "--dist",
+        "loadgroup",
         f"--numprocesses={args.num_processes}",
         "-x",
         "--durations",
         "0",
         "--timeout=2400",
         "test",
+        "-vv",
```

Comment:
This generates a lot more output - but when a test fails, it helps to have as much context as possible. It was difficult to diagnose some of the iOS failures with the low verbosity output.